pykern/pkcli/test.py

```diff
@@ -18,6 +18,7 @@
 
 SUITE_D = "tests"
 _COROUTINE_NEVER_AWAITED = re.compile("RuntimeWarning: coroutine .+ was never awaited")
+_TEST_SKIPPED = re.compile(".+ SKIPPED \(.+\)")
```

Comment:
If you'd like to match more securely, include line start/end, and don't forget `r`:

`re.compile(r"^.+\s+SKIPPED\s+\(.+\)$", flags=re.MULTILINE)`

Also be loose with spaces on parsing. If the output changes to two spaces or tab or something else, `\s` is more robust.